lib/ecto/type.ex

```diff
@@ -568,10 +553,6 @@ defmodule Ecto.Type do
   the given `loader` function is used.
   """
   @spec load(t, term, (t, term -> {:ok, term} | :error)) :: {:ok, term} | :error
-  def load({:embed, embed}, value, loader) do
-    load_embed(embed, value, loader)
-  end
```

Comment:
Should not we have an equivalent for `{:parameterized, type, opts}` match?

Reply:
It is handled by [this line](https://github.com/elixir-ecto/ecto/pull/3339/files/45a20ad9ca1e95b585265b5bf3aba18365150e20#diff-68b1275ffa052929bf96ca32ea92092dL561) which uses [this](https://github.com/elixir-ecto/ecto/pull/3339/files/45a20ad9ca1e95b585265b5bf3aba18365150e20#diff-68b1275ffa052929bf96ca32ea92092dR599)